Jenkinsfile

```diff
@@ -45,7 +35,7 @@ dockerizedBuildPipeline(
   //     }
   //   }
   // },
-  testResults: [ 'test-results.xml' ],
```

Comment:
Just removed while trying to get `make test` to work without capturing junit results.